operations/node_management/Add_a_Standard_Rack_Node.md

```diff
@@ -539,15 +538,9 @@ Usually there are two `cray-cps-cm-pm` pods, one on `ncn-w002` and one on `ncn-w
 
     ```text
     services   cray-cps-75cffc4b94-j9qzf    2/2  Running   0   42h 10.40.0.57  ncn-w001
```

Comment:
Check if this is correct

Reply:
I check with Kazuya and verified it on a running system.